registry/api/v2/names_test.go

```diff
@@ -149,3 +149,143 @@ func TestRepositoryNameRegexp(t *testing.T) {
 		}
 	}
 }
+
+func TestRepositoryNameRegexp(t *testing.T) {
+	for _, testcase := range []struct {
+		input   string
+		invalid bool
+	}{
+		{
+			input: "short",
```

Comment:
No need to repeat this list from the other test. Just use the presence of an error to infer validity.